specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -46,7 +46,39 @@ suppressions:
     reason: Suppress as instructed, this patch is for a abstract class and the type-discriminator needs to be required.
     where:
       - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/connections/{connectionName}"].patch.parameters[5].schema.properties.properties
-```
+  - code: ResourceNameRestriction
```

Comment:
Do you enforce any validation on resource names on the server side? Because if you do, then its basically good practice to document them in your swagger, and its not a real breaking change (and can be approved)

Reply:
Yes I think it's actually enforced by the service but never got reflected in the swagger, let me add the pattern instead

Reply:
@TimLovellSmith after confirming with the service engineer, we seems to have different naming restriction for different "kinds" of compute here, what do you suggest we should put in this case for the pattern restriction?

it's been documented here:https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsoftmachinelearningservices

Reply:
In that case we could just set a very general pattern, maybe to avoid the linter error, or suppress it, and link the docs on allowed names from the property description.